### PR TITLE
Make rpdtracer import only when required

### PR DESCRIPTION
**Pull request by Rohan138**
Patch for #208 to make RPD imports optional

### CODE REVIEW

**Comment by charlifu** [comment]
Please run `python3 -m pip install -r requirements-lint.txt && bash format.sh` to fix the linter error.

File: vllm/utils.py [hunk 1]
```diff
@@ -238,10 +238,10 @@ def __init__(self, name=None):
         self.name = name
 
     def __call__(self, func):
-        from hipScopedMarker import hipScopedMarker
-
+        
         @wraps(func)
         def inner(*args, **kwds):
+            from hipScopedMarker import hipScopedMarker
```
Comment (gshtras) [inline vllm/utils.py:244]: This is still being called when running vllm, with or without profiling in mind

**Comment by charlifu** [comment]
https://github.com/ROCm/vllm/blob/main/vllm/worker/worker.py#L27

**Comment by Rohan138** [comment]
@charlifu I believe we need the other rpd_mark imports, but they will now no-op if the required library isn't installed.

@AdrianAbeyta could you take a look?

**Review by gshtras — APPROVED** [review]

**Pull request merged**